pkg/proxy/healthcheck/healthcheck.go

```diff
@@ -343,5 +342,5 @@ func (h healthzHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 	} else {
 		resp.WriteHeader(http.StatusOK)
 	}
-	fmt.Fprintf(resp, fmt.Sprintf(`{"lastUpdated": %q,"currentTime": %q}`, lastUpdated, currentTime))
+	fmt.Fprint(resp, fmt.Sprintf(`{"lastUpdated": %q,"currentTime": %q}`, lastUpdated, currentTime))
```

Comment:
Same issue with fprint

Reply:
I've changed this to remove the `fmt.Sprintf` call instead :slightly_smiling_face:

Reply:
I think the main pint fo vllry was that `fmt.Sprintf` shouldn't be required here e.g.:

```
fmt.Fprintf(resp, `{"lastUpdated": %q,"currentTime": %q}`, lastUpdated, currentTime)
```

This has the same effect without but we don't generate string and pass it then to `fmt.Fprint`

Reply:
That's right, and I had already changed it into that. This diff is showing an older commit.